app/src/main/res/raw/list.txt

```diff
@@ -13,23 +13,23 @@ QQMusic,com.tencent.qqmusic,
 QQmail,com.tencent.androidqqmail,
 QQ English,com.qqeng.online,
 QQ 515,com.selfip.indigotv,
-iQIYI Video – Dramas & Movies,com.iqiyi.i18n,
+iQIYI Video,com.iqiyi.i18n,
 WPS Office Lite,cn.wps.moffice_i18n,
-WPS PDF - Free For PDF Read, View, Edit, Convert,cn.wps.pdf,
-WPS PDF Fill & Sign - Fill & Sign on PDF,cn.wps.pdf.fillsign,
+WPS PDF,cn.wps.pdf,
+WPS PDF Fill & Sign,cn.wps.pdf.fillsign,
 WPS Office Extra Goodies,cn.wps.moffice_extra,
 WPS Premium Subscription,cn.wps.moffice_premium,
 Bestie - Camera360 Beauty Cam,us.pinguo.selfie,
-Camera360: Selfie Photo Editor with Funny Sticker,vStudio.Android.Camera360,
+Camera360:,vStudio.Android.Camera360,
```

Comment:
remove that colon also, no need in this one

Reply:
ok